### DISCUSSION TITLE
[Showcase]Sunny Monday Labs

### DISCUSSION THREAD

**Discussion by bonskki** [gethyas/hyas#463] [Showcase]
### Site name

Sunny Monday Labs

### Public URL

https://sunnymonday.co.id/

### Short description

Sunny Monday Labs is an open-source-first company providing engineering excellence service.

### Full description

We are Sunny Monday Labs (Registered as PT Senin Sukses Sejahtera in Indonesia). Sunny Monday Labs is an open-source-first company providing engineering excellence service. We are based in Jakarta, Indonesia, known as the Emerging Tech City in Asia. Our main technology stack is [.NET technologies](https://dotnet.microsoft.com/en-us/platform/why-choose-dotnet) from Microsoft for its reliability and security. [Test-Driven Development (TDD)](https://martinfowler.com/bliki/TestDrivenDevelopment.html) is our main principle, helping us build a better web ecosystem with quality and innovation.

### Terms

- [X] I agree to the [Terms of Service](https://gethyas.com/terms/).

**Comment by h-enk** [comment]
Thanks for sharing! https://getdoks.org/showcase/
